Sources/DataPipeline/Plugins/Context.swift

```diff
@@ -1,14 +1,18 @@
 import CioInternalCommon
 import Segment
 
-/// Plugin class that adds device attributes to Segment requests
-class DeviceAttributes: Plugin {
+/// Plugin class that update the context properties in the request payload
+class Context: Plugin {
     public let type = PluginType.before
     public weak var analytics: Analytics?
 
-    public var token: String?
+    public var deviceToken: String?
     public var attributes: [String: Any] = [:]
 
+    var userAgentUtil: UserAgentUtil {
+        DIGraphShared.shared.userAgentUtil
+    }
```

Comment:
I would prefer it as `let` and initializing in constructor so it is assigned only once 🤓